mixnode/src/commands/run.rs

```diff
@@ -167,21 +161,25 @@ pub fn execute(matches: &ArgMatches) {
         "Announcing the following address: {}",
         config.get_announce_address()
     );
+    // TODO: Currently layer is always None in the mixnode and should be
+    // extracted at some point after the bonding process, by querying a
+    // validator
+    if let Some(layer) = config.get_layer() {
```

Comment:
Do we even need to store that in the config?

Reply:
Probably not, so I'm going to remove it and we'll see later if we need it or not.